tests/test_unwrap_no_args.py

```
@@ -68,17 +65,11 @@ def test_should_render_multiple_pages(page: Page):
-    page.wait_for_timeout(500)
     canvas_locator = pdf_viewer.locator("canvas")
-    canvas_list = wait_for_canvases(canvas_locator)
-
-    # Should have 8 pages total for the test PDF
-    assert len(canvas_list) == 8
+    expect(canvas_locator).to_have_count(8)
 
     # All canvases should be visible
-    for i, canvas in enumerate(canvas_list):
-        canvas.wait_for(timeout=5000, state='visible')
+    for i, canvas in enumerate(canvas_locator.all()):
```

Ruff (0.15.6)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)